modules/service/rest-api-v2.js

```diff
@@ -248,6 +268,81 @@ class RestAPIServiceV2 {
                 response: res,
             });
         });
+
+        /** Get root hash for provided data query
+         * @param Query params: data_set_id
+         */
+        server.get(`/api/${this.version_id}/fingerprint`, (req, res) => {
+            this.logger.api('GET: Fingerprint request received.');
+
+            const queryObject = req.query;
+            emitter.emit('api-get_root_hash', {
+                query: queryObject,
+                response: res,
+            });
+        });
+
+        server.get(`/api/${this.version_id}/import_info`, async (req, res) => {
+            this.logger.api('GET: import_info.');
+
+            const queryObject = req.query;
+            if (queryObject.data_set_id == null) {
+                res.send({ status: 400, message: 'Missing parameter!', data: [] });
+                return;
+            }
+
+            this.emitter.emit('api-import-info', {
+                dataSetId: queryObject.data_set_id,
+                responseFormat: queryObject.format || 'otjson',
+                response: res,
+            });
+        });
```

Comment:
This is one api route I think we should no longer support, since we added `get_dataset_info` route, and especially since this route effectively executes an export, which should be non blocking

Reply:
This route are used in some bdd tests, if we want to remove it we should update tests also. I think that it should be part of another task and not this one.